FlowCrypt/Assets.xcassets/edit.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Normally, similar icons would already be present from standard iOS icons. Could you check there first? So we wouldn't have to add our own.